### PR TITLE
CORE-19436: Provision StateManager instance for flow status in the REST worker

### PR DESCRIPTION
**Pull request by josephzunigadaly**

### CODE REVIEW

File: charts/corda/values.yaml [hunk 1]
```diff
@@ -1530,6 +1538,30 @@ workers:
...
+#          minSize: null
+#          # -- maximum time (in seconds) a connection can stay idle in the pool; a value of 0 means that idle connections are never removed from the pool
+#          idleTimeoutSeconds: 120
+#          # -- maximum time (in seconds) a connection can stay in the pool, regardless if it has been idle or has been recently used; if a connection is in-use and has reached "maxLifetime" timeout, it will be removed from the pool only when it becomes idle
+#          maxLifetimeSeconds: 1800
+#          # -- interval time (in seconds) in which connections will be tested for aliveness; connections which are no longer alive are removed from the pool; a value of 0 means this check is disabled
+#          keepAliveTimeSeconds: 0
+#          # -- maximum time (in seconds) that the pool will wait for a connection to be validated as alive
+#          validationTimeoutSeconds: 5
 #     TODO-[CORE-19372]: remove children and uncomment above 'keyRotation' section
```
Comment (jujoramos) [inline charts/corda/values.yaml]:
```suggestion
#     TODO-[CORE-19372]: remove children and uncomment above 'keyRotation' and 'flowStatus' sections
```

**Review by jujoramos — COMMENTED** [review]
Is the `rest-worker` the only worker type that needs to interact with the new `flowStatus` state type?.

**Comment by josephzunigadaly** [comment]
> Is the `rest-worker` the only worker type that needs to interact with the new `flowStatus` state type?.

Yes. It will be responsible for reading and writing to the new type.

**Review by jujoramos — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**